### DISCUSSION TITLE
Victron MPPT- is it normal such low output for 48V battery?

### DISCUSSION THREAD

**Discussion by greymda** [hoylabs/OpenDTU-OnBattery#1117] [Q&A] [answered]
So i connected my Victron 150/35 to my setup and for wjhatever reason when I connect my battery my breaker disconnects it..
Moreover the output of the MPPT seems odd? 14v?

<img width="1763" alt="image" src="https://github.com/user-attachments/assets/8b9ea10e-1676-470d-b3ae-bd5138085878">

Can you guys comment, I am lost of ideas...

**Comment by schlimmchen** [comment, marked as answer]
Well, that SmartSolar MPPT can be configured for different battery voltages. I guess you setup the wrong one. There is a physical dial on the MPPT to select the battery voltage, but that dial is ignored if the respective settings a made through the Bluetooth App. Please connect to your Victron SmartSolar MPPT using Bluetooth and the respective App and check the settings. If this is not a SmartSolar, read the manual and set the physical dial.

**Reply by Manos1966** [reply]
Thanks @schlimmchen added to Wiki 👍
